flocker/volume/service.py

```diff
@@ -216,30 +183,76 @@ def wait_for_volume(self, name):
 
         Polls the storage pool for the specified volume to appear.
 
+        XXX: Poll Openstack  API until block is unattached.
+
```

Comment:
Remove this comment and update docstring to explain the steps that are now performed here.